Dockerfile

```diff
@@ -54,6 +56,9 @@ RUN uname -a &&\
 # Copy in the binary
 COPY --from=builder /src/build/celestia /bin/celestia
 COPY --from=builder /src/./cel-key /bin/cel-key
+COPY --from=builder /src/./cel-shed /bin/cel-shed
+
+RUN cel-shed --help
```

Comment:
why is this necessary to have in the dockerfile?

Reply:
`RUN cel-shed --help` was only for debugging, removed it

We use `cel-shed` in our alerting system to get notified if all hardcoded bootstrappers are down simultaneously.
We would like to use the official container to automatically have the hardcoded list.